File: app/src/main/java/com/chinachino/mvvm/Testing/Testing.java
package com.chinachino.mvvm.Testing;

import android.util.Log;
//
import com.chinachino.mvvm.models.Result;

import java.util.List;

public class Testing {
    public static void Test(List<Result> results, String tag){
        for (Result result : results)
            Log.d(tag, tag +" : " +result.getTitle());
    }
}

File: app/src/main/java/com/chinachino/mvvm/testing/Testing.kt
package com.chinachino.mvvm.testing

import android.util.Log
import com.chinachino.mvvm.models.Result

//
object Testing {
    fun test(results: List<Result>, tag: String) {
        for (result in results) Log.d(tag, tag + " : " + result.title)
    }
}